StRoot/StETofCalibMaker/StETofCalibMaker.cxx

```diff
@@ -1674,9 +1674,9 @@ StETofCalibMaker::calculatePulserOffsets( std::map< unsigned int, std::vector< u
     double referenceTime = 0.;
 
     // update reference time (for QA)
+  if( pulserTimes.count( mReferencePulserIndex ) ) {
+    referenceTime = pulserTimes.at( mReferencePulserIndex ); //only updated for QA?? needed to remove smeared pulsers
     if( mDoQA ) {
```

Comment:
Philipp, maybe this line can be removed to simplify the code?
Question: does this affect the existing production data? e.g. Run20 FXT datasets which presumably have the ETOF calibrations in.